pom.xml

```diff
@@ -65,7 +65,7 @@
     <version.org.xhtmlrenderer>9.1.15</version.org.xhtmlrenderer>
     <version.bouncycastle>1.67</version.bouncycastle>
     <version.xmlsec>1.5.1</version.xmlsec>
-    <version.google.guava>29.0-jre</version.google.guava>
+    <version.google.guava>${version.com.google.guava}</version.google.guava>
```

Comment:
why not just remove the guava dep from the dependencyManagement section of the pom.xml and leave it the one managed at KIE parent? so we can ensure we're always aligned with the right version, wdyt?

Reply:
we could, even this doesn't change too much and we have still option to overwrite it here for any reason we need.